src/main/java/com/conveyal/datatools/manager/jobs/FeedUpdater.java

```diff
@@ -91,36 +95,31 @@ public void run() {
             LOG.info("Running initial check for feeds on S3.");
             eTagForFeed = new HashMap<>();
         }
+        LOG.info("Checking for feeds on S3.");
         Map<String, String> newTags = new HashMap<>();
-        // iterate over feeds in download_prefix folder and register to gtfsApi (MTC project)
+        // iterate over feeds in download_prefix folder and register to (MTC project)
         ObjectListing gtfsList = FeedStore.s3Client.listObjects(feedBucket, bucketFolder);
+        LOG.info(eTagForFeed.toString());
         for (S3ObjectSummary objSummary : gtfsList.getObjectSummaries()) {
 
             String eTag = objSummary.getETag();
+            String keyName = objSummary.getKey();
+            LOG.info("{} etag = {}", keyName, eTag);
             if (!eTagForFeed.containsValue(eTag)) {
```

Comment:
This `eTagForFeed` map variable never uses the key except possibly for logging/debugging purposes.  Consider making this variable a Set instead.

Reply:
I have found it helpful to have this as a map during debugging/logging, so I think we should keep it that way.